mlflow/tracking/fluent.py

```diff
@@ -1977,8 +1966,8 @@ def get_autologging_params(autolog_fn):
 
     def setup_autologging(module):
         try:
-            autolog_fn = LIBRARY_TO_AUTOLOG_FN[module.__name__]
-
+            autologging_params = None
+            autolog_fn = eval(LIBRARY_TO_AUTOLOG_FN[module.__name__])  # noqa: S307
```

Comment:
Can we use `importlib.import_module` instead of `eval`?

Reply:
yea, that is doable. Pushed a change.